apps/accounts/templates/base.html

```diff
@@ -7,6 +7,62 @@
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		{%- comment -%} Load sentry before any CSS/other JS loads {%- endcomment -%}
```

Comment:
Would you change the  `{%- comment -%}` to be `{% comment %}` ?

I think this `{%-` formulation might be used templating languages in nunjucks, but unfortunately it's not supported in django's own templating language